inspire_schemas/records/seminars.yml

```
@@ -130,6 +133,12 @@ properties:
             Contains a reference to the record replacing the current one, if it
             is marked as :ref:`deleted`.
         title: Record replacing this one
+    material:
+        items:
+            $ref: elements/url.json
+        title: Links to PDFs or slides.
```

Slides can be PDFs but don't have to:
```suggestion
        title: Links to related materials such as slides.
```
To me it's not clear if we want only slides here or also potentially other stuff.